Grammar.md

```diff
@@ -28,9 +28,21 @@ The required grammatical case should be specified right in instruction's substit
 - Instruction text formatter ([index.js](index.js) in this module) should:
   - check `{way_name}` and `{rotary_name}` variables for optional grammar case after colon: `{way_name:accusative}`
   - find appropriate regular expressions block for target language and specified grammar case
-  - call standard [string replace with regular expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) for each expression in block passing result from previous call to the next; the first call should enclose original street name with whitespaces to make parsing words in names a bit simplier.
+  - call standard [string replace with regular expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) for each expression in block passing result from previous call to the next; the first call should enclose original street name with whitespaces to make parsing several words inside name a bit simplier.
```

Comment:
Is this still a requirement, now that `grammarize` is called before capitalization and insertion (Project-OSRM/osrm-text-instructions#170)?